Controllers/HotelRoomController.cs

```diff
@@ -33,13 +33,13 @@ public void Post([FromBody] string value)
         }
 
         // PUT api/<HotelRoomController>/5
-        [HttpPut("{id}")] // /api/Hotels/{hotelId}/Rooms/{roomNumber
+        [HttpPut("{roomNumber}")] // /api/Hotels/{hotelId}/Rooms/{roomnumber}
         public void Put(int id, [FromBody] string value)
```

Comment:
Parameters here need to include `hotelId` and `roomNumber`, and we need a model with additional data (e.g. `PetFriendly`).
```suggestion
        public void Put(int hotelId, int roomNumber, [FromBody] CreateHotelRoom model)
```